File: src/main/java/yuu/Season_command.java
package yuu;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import sun.plugin2.main.server.Plugin;

public class Season_command implements CommandExecutor {

    CustomConfig season;
    private Main plugin;

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        season = new CustomConfig(plugin, "season.yml");


        return false;
    }
}

File: src/main/java/yuu/Season_command.kt
package yuu

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import sun.plugin2.main.server.Plugin

class Season_command internal constructor(private val plugin: Main) : CommandExecutor {

    private val season: CustomConfig = CustomConfig(plugin, "season.yml")

    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {


        return false
    }
}
